source/release-notes/backend.txt

```diff
@@ -13,6 +13,40 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240710:
+
+10 July 2024 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- Edge Server
+
+  - The bootstrap progress percentage is now shown in the Edge Server status UI.
+
+- :ref:`Enabling automatic deployment <deploy-github>` allows App Services to automatically push a copy of your App to your designated repository, 
+  making the current :guilabel:`Export your application and upload to GitHub` section in the Deployment UI unnecessary. Updated the description 
+  under the :guilabel:`Enable Automatic Deployment` section to make this behavior clear. 
+
+- Deployment logs for your application now record the ``user_id`` for Admin API and CLI requests.
+
+- Fixed bug where :ref:`match expressions <database-triggers-match-expression>` for Triggers were converted to incorrect formats if they contained a regular expression (``$regex``).
+
+- App Services Function names can now include hyphens (-).
+
+- When a free or shared Atlas cluster exceeds 500 collections, the max number of collections for these tiers, you will get an error message.
+  For more information on the operational limits of free or shared clusters, see the :atlas:`Free and Shared Cluster Operational Limits <reference/free-shared-limitations/#operational-limits>`.
```

Comment:
this link 404s for me. needs a slash before `<reference/...` maybe?

Reply:
Fixed!